examples/knowledge_generation_using_nemotron/utils.py

```
@@ -0,0 +1,38 @@
+    else:
+        knowl_train_pretrain = knowl_train.map(_conv_pretrain, fn_kwargs={"tokenizer": tokenizer}, num_proc=10)
+    if nemotron_format:
+        knowl_train_pretrain = knowl_train_pretrain.map(lambda x: {'messages': [{'content': f'detailed thinking {thinking}', 'role': 'system'}] + x['messages']})
+    if create_summary:
+        summary_ds = create_auxiliary_dataset(ds)
+        if no_pretrain and summary_ds:
+            summary_ds_pretrain = summary_ds
+        else:
+            summary_ds_pretrain = summary_ds.map(_conv_pretrain, fn_kwargs={"tokenizer": tokenizer}, num_proc=10)
+        if nemotron_format:
+            summary_ds = summary_ds.map(lambda x: {'messages': [{'content': 'detailed thinking off', 'role': 'system'}] + x['messages']})
```

Ruff (0.8.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)